pkg/cmd/pulumi/convert.go

```diff
@@ -346,19 +374,35 @@ func runConvert(
 		pCtx.Diag.Logf(sev, diag.RawMessage("", msg))
 	}
 
-	installProvider := func(provider tokens.Package) *semver.Version {
+	installPulumiProvider := func(provider tokens.Package) *semver.Version {
 		// If auto plugin installs are disabled just return nil, the mapper will still carry on
 		if env.DisableAutomaticPluginAcquisition.Value() {
 			return nil
 		}
 
+		// TODO debug this by printing packageBlockDescriptors
+		if p, ok := packageBlockDescriptors[string(provider)]; ok && p.Name == "terraform-provider" {
```

Comment:
I specifically here am just checking if we already generated pcl using terraform provdier so I can print warnings to the user and have diags for debugging if stuff goes wrong.

I think I shouldn't exit though, since we still need to install the dynamic bridge if it it is not present.

Reply:
I think this whole block should go. The warning about no parameterisation is a warning for the terraform provider to decided on, and we don't need to warn the user about every local sdk we build. They should just happen and be expected.